plugins/katello/3.11/user_guide/provisioning/index.md

```diff
@@ -21,3 +21,27 @@ Katello ships a number of templates in addition to the standard Foreman ones. Wh
 To customize any of the above templates, simply clone them and add your changes.
 
 When you synchronize a repository with a distribution such as Fedora or CentOS, Katello will automatically create the operating system and assign these default templates.  You may change the defaults by going to Administer > Settings, and selecting the Katello tab.
+
+If provisioning hosts using a synced CentOS 8 repository, the AppStream repository is needed for Kickstart to work.  To set this up, create an "AppStream" repository within the same lifecycle environment and content view as the CentOS 8 repository and sync it.  Katello will automatically add this AppStream repo to the Kickstart file.  After creating the host using CentOS 8 content, double check that the generated Kickstart file mentions the AppStream repo.  The host provisioning should proceed automatically as expected.
+
+**Note:** Currently, CentOS 8 Anaconda requires that the AppStream repository is specifically named "AppStream".  To get around this, edit the Kickstart default provisioning template (or create a new one) to ensure this block:
```

Comment:
if there is an open centos8 issue for this, it would be nice to link here so people can quickly tell if this is still relevant

Reply:
(obviously, this goes for all versions updated)